tox.ini

```diff
@@ -61,7 +61,7 @@ commands =
     # codespell {[vars]lib_path}
     codespell {toxinidir} --skip {toxinidir}/.git --skip {toxinidir}/.tox \
       --skip {toxinidir}/build --skip {toxinidir}/venv \
-      --skip {toxinidir}/.mypy_cache --skip {toxinidir}/icon.svg
+      --skip {toxinidir}/.mypy_cache --skip {toxinidir}/icon.svg --skip {toxinidir}/*.log
```

Comment:
You shouldn't need this anymore, as per https://github.com/canonical/operator-workflows/pull/105

Reply:
Removed, thanks!